R2Northstar.sln

```diff
@@ -11,28 +11,66 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "loader_launcher_proxy", "lo
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "loader_wsock32_proxy", "loader_wsock32_proxy\loader_wsock32_proxy.vcxproj", "{CF55F3B5-F348-450A-9CCB-C269F21D629D}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "DiscordRPC", "DiscordRPC\DiscordRPC.vcxproj", "{2601AF91-0EB7-475E-AD32-01589A47F75E}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "DiscordGameSDK", "DiscordGameSDK\DiscordGameSDK.vcxproj", "{87BA15AA-96EE-47B8-9A10-1F41FA927AAD}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
```

Comment:
Extraneous build configs.

Reply:
They're still there.

Reply:
...

Reply:
To fix this, close VS, open the SLN file in notepad, then remove these lines. Alternatively, run `git checkout main R2Northstar.sln -f`, reload the solution, then commit the result.

Reply:
...

Reply:
ah fuck i forgot to do this